util/fiber_socket_base.h

```diff
@@ -93,29 +127,29 @@ class FiberSocketBase : public io::Sink, public io::AsyncSink, public io::Source
 
 class LinuxSocketBase : public FiberSocketBase {
  public:
-  using native_handle_type = int;
+  using FiberSocketBase::native_handle_type;
   constexpr static unsigned kFdShift = 4;
 
   virtual ~LinuxSocketBase();
 
-  native_handle_type native_handle() const {
+  native_handle_type native_handle() const override {
     static_assert(int32_t(-1) >> kFdShift == -1);
 
     return fd_ >> kFdShift;
   }
 
   /// Creates a socket. By default with AF_INET family (2).
-  error_code Create(unsigned short protocol_family = 2);
+  error_code Create(unsigned short protocol_family = 2) override;
 
-  ABSL_MUST_USE_RESULT error_code Bind(const struct sockaddr* bind_addr, unsigned addr_len);
-  ABSL_MUST_USE_RESULT error_code Listen(unsigned backlog);
+  ABSL_MUST_USE_RESULT error_code Bind(const struct sockaddr* bind_addr, unsigned addr_len) override;
+  ABSL_MUST_USE_RESULT error_code Listen(unsigned backlog) override;
 
   // Listens on all interfaces. If port is 0 then a random available port is chosen
   // by the OS.
-  ABSL_MUST_USE_RESULT error_code Listen(uint16_t port, unsigned backlog);
+  ABSL_MUST_USE_RESULT error_code Listen(uint16_t port, unsigned backlog) override;
 
   // Listen on UDS socket. Must be created with Create(AF_UNIX) first.
-  ABSL_MUST_USE_RESULT error_code ListenUDS(const char* path, mode_t permissions, unsigned backlog);
+  ABSL_MUST_USE_RESULT error_code ListenUDS(const char* path, mode_t permissions, unsigned backlog)override;
```

Comment:
formatting?

Reply:
That's really weird, 99.9% it's a clang-format bug, because I did run it before I pushed

Reply:
and speaking of that I will add pre-commit here.